services/app-api/src/postgres/prismaTypes.ts

```diff
@@ -15,6 +20,22 @@ const updateInfoIncludeUpdater = {
 
 type UpdateInfoTableWithUpdater = UpdateInfoTable & { updatedBy: User }
 
-export type { PrismaTransactionType, UpdateInfoTableWithUpdater }
+type ContractFormData = {
```

Comment:
Oh ok, this is pretty much the init stuff, maybe just rename this type to be InsertContractArgsType and go with that.